src/components/ChatBar.tsx

```diff
@@ -66,6 +67,33 @@ const ChatBar: React.FC = () => {
   const [openChatHistory, setOpenChatHistory] = useState(false);
   const [currentChatId, setCurrentChatId] = useState("");
 
+  const { theme } = useThemeContext();
+  const isDarkMode = theme.palette.mode === "dark";
+
+  const chatContainerStyle = {
+    backgroundColor: theme.palette.background.paper,
+  };
+
+  const messageListStyle = {
+    backgroundColor: theme.palette.background.paper,
+  };
+
+  const messageStyle = isDarkMode
+    ? {
+        backgroundColor: theme.palette.background.default,
+        color: theme.palette.text.primary,
+      }
+    : {};
+
+  const avatarStyle = {
+    filter: isDarkMode ? "invert(1)" : "none",
+  };
+
+  const typingIndicatorStyle = {
+    backgroundColor: theme.palette.background.paper,
+    color: theme.palette.text.primary,
+  };
```

Comment:
Not sure if there is a better way to apply the MUI themes, this was the best I could come up with, but let me know if you have another idea